src/System.Buffers/ref/project.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+      "System.Runtime": "4.4.0-beta-24625-01"
+    },
+  "frameworks": {
+    "netstandard1.1": {}
```

Comment:
I guess the reason is because perhaps the surface area didn't change compared to what we have before, it just looks weird that we didn't have a reference assembly before (since we were using the implementation as reference assembly) and now we do for the new framework but we are targetting netstandard1.7